lib/block-supports/layout.php

```diff
@@ -280,6 +280,37 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 				);
 			}
 		}
+	} elseif ( 'grid' === $layout_type ) {
+		$minimum_column_width = ! empty( $layout['minimumColumnWidth'] ) ? $layout['minimumColumnWidth'] : '12rem';
```

Comment:
Just a future proofing question: if `minimumColumnWidth` isn't set, this assumes a default of `12rem`. If we want to enable the control over individual columns again in follow-ups, how will we tell that we shouldn't be outputting `12rem`?

By which I mean, what are the trade-offs between using `12rem` as an inferred value, versus explicitly defining `12rem` within the Grid variation? For example, what if other blocks wished to use a different default minimum column width?

Reply:
It would be pretty easy to introduce something like `isResponsive` again, or any other property that indicates that the grid type is not based on column width (which would always be the default). We'll need a check before outputting the value because the whole template-columns declaration will be different for a static grid, and even if a `minimumColumnWidth` is set (e.g. if we don't reset it on switching to another grid type) we will only use the value if the grid is of the responsive type (or whatever we want to call it).

I chose 12rem arbitrarily, it could be any other value but we do need to have _some_ default value here or the `auto-fill` won't work.

If other blocks want to use a different value they can set it when they define the layout type, e.g. `layout: { type: 'grid', minimumColumnWidth: '9rem' }`.

Reply:
So (just to make sure I'm following), we're essentially saying, a grid layout with absolutely no other values will default to `12rem` minimum column width. For future changes, we're adding additional attributes that will be set, so it'll be easy to determine when to switch off the current logic. And for changing the `12rem` default on a block-by-block basis, we provide the default type type value as you mention.

Sounds solid to me! 👍

Reply:
Yeah, the idea is to have a sensible default so you can just add a grid and it'll look ok for the majority of cases? Then progressively add configurability.